### PR TITLE
UX: show lock icon in user stream if topic is closed/archived

### PR DESCRIPTION
**Pull request by techAPJ**
Example:

screen shot 2015-05-11 at 22 13 15

cc @coding-horror

### CODE REVIEW

File: app/serializers/user_action_serializer.rb [hunk 1]
```diff
@@ -77,4 +79,12 @@ def include_edit_reason?
     object.action_type == UserAction::EDIT
   end
 
+  def closed
+    Topic.find_by(id: object.topic_id).closed if object.topic_id.present?
```
Comment (ZogStriP) [inline app/serializers/user_action_serializer.rb]: Hmm, I can smell the `N+1` query here

**Comment by techAPJ** [comment]
@ZogStriP: Just updated the code. Looks better now?

**Comment by ZogStriP** [comment]
:+1:

**Comment by eviltrout** [comment]
Looks good!

**Pull request merged**